compiler-collection.js

```diff
@@ -0,0 +1,47 @@
+module.exports = (expireAfterSeconds) => {
+  const expireMs = expireAfterSeconds * 1000;
+  const compilers = new Map();
```

Comment:
`CompilerMap`. neat!

Reply:
Yeah, I want to get used to using the new ES5 classes. JS has lots more neat stuff now-a-days.